.github/labeler.yml

```
@@ -0,0 +1,57 @@
+zammad-ldap-sync:
+- changed-files:
+  - any-glob-to-any-file: /charts/zammad-ldap-sync/**
+
+refarch-gateway:
+- changed-files:
+  - any-glob-to-any-file: /charts/refarch-gateway/**
+
+refarch-gateway:
+- changed-files:
+  - any-glob-to-any-file: /charts/refarch-gateway/**
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)